docs/source/nextflow-workflow.rst

```diff
@@ -135,6 +135,143 @@ Example ``sample_sheet.csv``
     +-------------------------------------------------+------------------------------------------------------+---------------------------------------------------------------------+
 
 
+Quick start
+###################
+The following is a condensed summary of steps required to get Autometa installed, configured and running. There are links throughout to the appropriate documentation sections that can provide more detail if required.
+
+Installation
+*******************
+For full installation instructions, please `click here <https://autometa.readthedocs.io/en/latest/installation.html>`_
+
+Installation relies on Miniconda being installed on your system (link here: `<https://docs.conda.io/en/latest/miniconda.html>`_).
+To install the conda environment run the following code:
+
+.. code-block:: bash
+
+    conda env create --file=https://raw.githubusercontent.com/KwanLab/Autometa/main/nextflow-env.yml
+
+Next, activate the environment:
+
+.. code-block:: bash
+
+    conda activate autometa-nf
+
+Configuring a scheduler
+*******************
+For full details on how to configure your scheduler, please `click here <https://autometa.readthedocs.io/en/latest/nextflow-workflow.html?highlight=slurm#configuring-your-process-executor>`_
+
+If you are using a Slurm scheduler, you will need to create a configuration file. If you do not have a scheduler, skip ahead to "Running Autometa".
+
+First you will need to know the name of your slurm partition. Run :code:`sinfo` to find this. In the example below, the partition name is "queue".
+
+.. image:: ../img/slurm_partitions.png
+
+Next, generate a new file called slurm_nextflow.config via nano:
+
+.. code-block:: bash
+
+    nano slurm_nextflow.config
+
+Then copy the following code block into that new file:
+
+.. code-block:: bash
+
+    profiles {
+            slurm {
+                process.executor       = "slurm"
+                process.queue = "queue" // <<-- change this to whatever your partition is called
+                // queue is the slurm partition to use in our case
+                docker.enabled         = true
+                docker.userEmulation   = true
+                singularity.enabled    = false
+                podman.enabled         = false
+                shifter.enabled        = false
+                charliecloud.enabled   = false
+                executor {
+                    queueSize = 8
+                }
+            }
+        }
+
+Keep this file somewhere central to you. For the sake of this example I will be keeping it in a folder called "Useful scripts" in my home directory e.g. :code:`/home/me/Useful_scripts/slurm_nextflow.config`
```

Comment:
I've put my reasoning in the docs now, but I always recommend to the SA crew to keep important scripts in either Slurm-scripts (if it's a batch script) or a folder called "Useful scripts" for anything else that's useful and not linked to a particular project. Helps them keep things organized otherwise it all sits in the equivalent of "home/sam/" and takes on the form of an unorganized desktop very quickly :)